bundle-size/Tabster.fixture.js

```diff
@@ -0,0 +1,45 @@
+import {
+    createTabster,
+    disposeTabster,
+    forceCleanup,
+    getTabsterAttribute,
+    setTabsterAttribute,
+    getGroupper,
+    getMover,
+    getCrossOrigin,
+    getDeloser,
+    getModalizer,
+    getObservedElement,
+    getOutline,
+    getCurrentTabster,
+    getInternal,
+    overrideBasics,
+    makeNoOp,
+    isNoOp,
```

Comment:
```suggestion
    createTabster,
    disposeTabster,
    getTabsterAttribute,
    setTabsterAttribute,
    getGroupper,
    getMover,
    getDeloser,
    getModalizer,
    getCurrentTabster,
```

I would start with this, it' s what is intended to be widely used by our partners

Reply:
instead, I'll create another fixture, It would still be good to have one for all we export IMO